src/RatkoR/Crate/Schema/Blueprint.php

```diff
@@ -25,6 +26,14 @@ class Blueprint extends \Illuminate\Database\Schema\Blueprint {
      */
     protected $indexes = [];
 
+    /**
+     * This Array is used to partition the Crate Table into multiple parts.
+     * This is a advanced feature in create and should be used with caution
+     *
+     * @var array
+     */
+    protected $partionedBy = [];
```

Comment:
Was this supposed to be `$partitionedBy`? (missing the extra `ti` in variable name)